.appveyor.yml

```diff
@@ -59,28 +68,30 @@ install:
   - ps: Set-Location -Path $env:APPVEYOR_BUILD_FOLDER
 
   # Verify CWD
-  - ps: Write-Host "[+] Current working directory is ..."
+  - ps: Write-Host "[+] Current PowerShell directory is ..."
   - ps: Get-Location
   - bash -e -l -c "cd $APPVEYOR_BUILD_FOLDER && ls -alh"
 
-  # Install Python modules
-  - ps: python -m pip install --upgrade pip
-  # - ps: python -m pip install --upgrade packager
-  - ps: python -m pip install --upgrade pyinstaller
-
   # Install NPM modules
-  # - bash -e -l -c "cd /C/Tridactyl && npm install -g windows-build-tools"
   - bash -e -l -c "cd $APPVEYOR_BUILD_FOLDER && npm install"
 
 build_script:
   # Add Python-3.6 to %PATH%
   - ps: $env:PATH = "C:\Python36-x64\Scripts;$env:PATH"
   - ps: $env:PATH = "C:\Python36-x64;$env:PATH"
-  - ps: Copy-Item -Path "C:\Python36-x64\Python.exe" -Destination "C:\Python36-x64\Python3.exe"
+  - ps: >-
+        Copy-Item `
+        -Path "C:\Python36-x64\Python.exe" `
+        -Destination "C:\Python36-x64\Python3.exe"
+  - python --version
+
+  # Install Python modules under Python 3.6
+  - ps: python -m pip install --upgrade pip
```

Comment:
Python 3.4+ should include pip by default - do you have reason to believe that it doesn't?

Reply:
> do you have reason to believe that it doesn't?

No. 

I was doing the `--upgrade` mostly to get the latest `10.0.1` release. That's the version I have locally installed. 

We should probably "pin" the version numbers in here to be more clean and pedantic. Let me add them in. In case something breaks in the future, it'll be easier to spot.

Thanks.